pkgs/tools/misc/radeon-profile/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, qt5, xorg }:
```

Comment:
From the nixpkgs manual (http://nixos.org/nixpkgs/manual/#ssec-qt-applications):

> Dependencies should be imported unqualified, i.e. qtbase not qt5.qtbase. Do not import a package set such as qt5 or libsForQt5 into your package; although it may work fine in the moment, it could well break at the next Qt update.